mods/ra/rules/vehicles.yaml

```diff
@@ -714,6 +714,12 @@ QTNK:
 	SelectionDecorations:
 		VisualBounds: 44,38,0,-4
 	MadTank:
+	Detonates:
+		ChargeDelay: 96
+		DetonationDelay: 42
+		ChargeSound: MADCHRG2.AUD
+		DetonationSound: MADEXPLO.AUD
```

Comment:
I'm not going to change this. Someone else can pick it up as a part of a greater cleanup if it means that much to them.